### ISSUE TITLE
[Bug Report]

### ISSUE THREAD

**Issue by GitTrollAT** [SapuSeven/BetterUntis#254]
<details>
<summary>Logs</summary>

```
Anfragen-ID: 1
Fehlercode: 3
Fehlermeldung: o7.b: Field 'staffInfo' is required for type with serial name 'com.sapuseven.untis.models.untis.timetable.PeriodText', but it was missing
```
</details>

**Additional information**

- Android version: _11_
- BetterUntis version: _3.4.7 (31)_
- Installation source: _com.android.vending_

**Comment by Pxddyk45** [comment]
I have the same Issue

**Comment by lm41** [comment]
This has been fixed in version 3.4.8. You can opt-in as a beta tester on Google Play to receive the update sooner.
